chain/network/src/stats/rate_counter.rs

```diff
@@ -37,7 +40,8 @@ impl RateCounter {
     /// Increment number of bytes transferred, updating counts and rates.
     pub fn increment(&mut self, bytes: u64) {
         let now = millis_since_epoch();
-        self.entries.push_back(Entry { bytes, expiration_timestamp: now + MINUTE_IN_MILLIS });
+        self.entries
+            .push_back(Entry { bytes, expiration_timestamp: now + MINUTE_IN_MILLIS.as_millis() });
```

Comment:
meta comment:

seems that this class (RateCounter) might not work properly if there is no data coming in (as then truncate is not called and bytes_per_min / count_per_min show results from the past).

we should add a clear warning about it to the class comment (and check if this can cause us trouble).

Alternatively, we could fix it, by having a bytes_per_min/count_per_min actually call the truncate - if truncate wasn't called recently.

Reply:
I think it's correct. I changed how `expiration_timestamp` is used to represent current time, and make the logic less confusing.

Reply:
Fixed in https://github.com/near/nearcore/pull/5225/files